scopes/dependencies/modules/packages-excluder/generate-node-modules-pattern.ts

```diff
@@ -23,6 +23,10 @@ type GenerateNodeModulesPatternOptions<T> = {
    * The target for which patterns are generated.
    */
   target?: T;
+  /**
+   * Defines if pnpm is enabled as a package manager
+   */
+  isPnpmEnabled: boolean;
```

Comment:
I think we should make this optional (default true)

Reply:
if it's not optional, people might get now type errors if they update the package excluder but not bit